rkt/run.go

```diff
@@ -256,6 +259,18 @@ func runRun(args []string) (exit int) {
 		return 1
 	}
 
+	if flagInteractive {
+		ac, err := p.getAppCount()
+		if err != nil {
+			stderr("run: cannot get container's app count: %v", err)
+			return 1
+		}
+		if ac > 1 {
+			stderr("run: interactive option only supports containers with one app")
+			return 1
+		}
+	}
```

Comment:
Not really, this is the case where `--interactive` and `--pod-manifest` are both set. But good question, should we enable both?